packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -372,3 +396,44 @@ class VideoPlayerOptions {
   /// currently no way to implement this feature in this platform).
   final bool mixWithOthers;
 }
+
+/// Subtitle option which embedded into video.
+class EmbeddedSubtitle {
+  /// Subtitle option which embedded into video.
+  ///
+  /// * It's recommended to not create direct instance by this constructor. Try to
+  ///   get from video player controller.
```

Comment:
It's recommended not to create a direct instance with this constructor. Try to get it from the video player controller.